tests/drivers/gpio/gpio_api_1pin/testcase.yaml

```diff
@@ -1,8 +1,14 @@
+common:
+  tags: drivers gpio
+  depends_on: gpio
+
 tests:
   peripheral.gpio.1pin:
-    tags: drivers gpio
-    depends_on: gpio
     min_flash: 48
     # Fix exclude when we can exclude just sim run
     platform_exclude: mps2_an385 mps2_an521 neorv32
     filter: dt_enabled_alias_with_parent_compat("led0", "gpio-leds")
+
+  peripheral.gpio.bd8lb600fs:
+    platform_allow: native_posix
+    extra_args: DTC_OVERLAY_FILE="boards/gpio_bd8lb600fs.overlay"
```

Comment:
How is this supposed to work on native posix?

Reply:
Good point, I am actually uncertain what I should put there, Because there is no board available with this hardware soldered onto it, I just have my test setup which I built for myself.

Reply:
You could add a generic shield definition using this chip. That what I have done in the past.

Reply:
Could you maybe point me to an example as a starting point?

Reply:
One possible source of inspiration could be https://github.com/zephyrproject-rtos/zephyr/tree/main/boards/shields/ssd1306

Reply:
Thanks, I figured out you used this shield for the display driver sample.

So what I could do would be to add this shield and use it for this testcase. But I assume nobody will every specifically add a test with hardware in the loop for exactly this setup. Therefore, I don't  think it is worth it to add this test. I will remove it again.